app/templates/_base_page.html

```diff
@@ -3,12 +3,35 @@
 {# Import GOV.UK Frontend components for globale usage#}
 {% from "components/breadcrumbs/macro.njk" import govukBreadcrumbs %}
 {% from "components/button/macro.njk" import govukButton %}
+{% from "components/phase-banner/macro.njk" import govukPhaseBanner %}
 {% from "components/tabs/macro.njk" import govukTabs %}
 
+{# This can be removed once we start using govuk-frontend template #}
+{# The reason calling this block here is to change the header title so that it does not
+   include the phase tag #}
+{% block inside_header %}
+  <div class="header-title">
+    <a href="/">Digital Marketplace</a>
+  </div>
+{% endblock %}
+
+{# This can be removed once we start using govuk-frontend template #}
+{# The reason for calling this is to ensure we do not output the user research signup banner #}
+{% block after_header %}
+{% endblock %}
+
 {# override content block to add a beforeContent block like the one in govuk-frontend template #}
 {% block content %}
   {% block top_header %}{% endblock %}
   <div id="wrapper">
+    {# This is only a temporary place for the phase banner until we start using govuk-frontend template #}
+    {{ govukPhaseBanner({
+      "tag": {
+        "text": "beta"
+      },
+      "html": 'Help us improve the Digital Marketplace - <a class="govuk-link" href="https://www.digitalmarketplace.service.gov.uk/help">send your feedback</a>'
```

Comment:
Can we use the `url_for()` here instead of the hardcoded link - See https://github.com/alphagov/digitalmarketplace-frontend-toolkit/blob/master/toolkit/templates/proposition-header.html#L10 

This way it will work for preview and staging as well.

Reply:
Thanks @katstevens, I'll keep that in mind for any future urls.